ssha/config.py

```diff
@@ -166,6 +199,10 @@ def load(name):
         proxy_command = ['ssh'] + known_hosts_options + ['-W', '%h:%p', '${bastion.address}']
         add('ssh.proxy_command', ssh.format_command(proxy_command))
 
+    # Generate a temporary key if specified
```

Comment:
Please make this comment look like the other computed values - `To support configs like this:` etc.